test-suite/js/src/test/require-modules/org/scalajs/testsuite/jsinterop/ModulesTest.scala

```diff
@@ -197,11 +177,11 @@ object ModulesTest {
   class NonExistentSuperClass extends js.Object
```

Comment:
Should we adapt the `@JSImport` on this class to be `non-existent.js` instead of `non-existent`?